lightTheme.ts

```diff
@@ -0,0 +1,163 @@
+import { createTheme } from '@mui/material/styles';
+
+import { inter } from './constants/inter-latin';
+
+export const themeTypography = {
+  fontFamily: inter.style.fontFamily,
+  h1: {
+    fontSize: '3.5em',
+    fontWeight: '700',
```

Comment:
Values without units shouldn't be quoted:
`fontWeight: '700',` -> `fontWeight: 700,`

Reply:
'+